html/functions.php

```diff
@@ -49,21 +49,46 @@ function createTweet($text, $user_id)
     return $stmt->execute();
 }
 
+function createReTweet($text, $reply_post_id,$user_id)
```

Comment:
```suggestion
function createReTweet($text, $reply_post_id, $user_id)
```

「,」の後ろにスペースをいれてください
1つのPJないでは、全て体裁を同じにします。

（スペースがあったり、なかったりすると読みにくいし、なにか意図があってそうしているのか？となるため）